### DISCUSSION TITLE
After update to version 0.21.1 , I got a error `No .csproj file found`

### DISCUSSION THREAD

**Discussion by marsen** [stryker-mutator/stryker-net#1384] [Q&A] [answered]
The error message in my github actions is blew 
For more, can see it on https://github.com/marsen/Marsen.NetCore.Dojo/runs/1760128678?check_suite_focus=true

 Version: 0.21.1 (beta)

[06:57:56 WRN] Stryker is running in single threaded mode due to max concurrent testrunners being set to 1.
[06:57:56 INF] Identifying project to mutate.
[06:57:56 INF] Time Elapsed 00:00:00.1929827

No .csproj file found, please check your project directory at /home/runner/work/Marsen.NetCore.Dojo/Marsen.NetCore.Dojo

Error: Process completed with exit code 1.

**Comment by rouke-broersma** [comment, marked as answer]
Hi @marsen 

This sounds like a bug, so I created an issue for you here #1387
